automation/test.sh

```diff
@@ -29,6 +29,10 @@ export CDI_NAMESPACE=$NAMESPACE
 
 echo "namespace: ${NAMESPACE}, cdi-namespace: ${CDI_NAMESPACE}"
 
+if [[ -n "$RANDOM_CR" ]]; then
```

Comment:
does anything ever set RANDOM_CR?

Reply:
I am going to set it in project-infra as soon as I am done getting this out the door. Probably in the ceph lane.

Reply:
https://github.com/kubevirt/project-infra/pull/645